maven_iuvs/instrument.py

```diff
@@ -239,3 +243,67 @@ def mcp_volt_to_gain(volt, channel="FUV"):
         gain = A*np.exp(alpha*volt)
 
     return gain
+
+
+# TODO: Move these to appropriate places once done.
+def convert_spectrum_DN_to_photons(light_fits, spectrum):
+    """
+    Converts a spectrum in DN to photo events. 
+
+    Parameters
+    ----------
+    light_fits : astropy.io.fits instance
+                 File with light observation
+    spectrum : array
+               A spectrum recorded in DN. May be a fitted LSF or actual data.
+
+    Returns
+    ----------
+    spectrum converted ot photoevents.
+    """
+    
+    return spectrum * DN_to_PE_conversion_factor(light_fits)
+
+
+def DN_to_PE_conversion_factor(light_fits):
+    """
+    For a given light observation file, this returns the conversion factor which, when multiplied by a spectrum in DN/sec/nm,
+    will convert the spectrum to photons.
+
+    Parameters
+    ----------
+    light_fits : astropy.io.fits instance
+                 File with light observation
+
+    Returns
+    ----------
+    conv_factor : float
+                  Multiplies a spectrum in DN to convert it to photoevents.
+    """
+    gain = light_fits['Engineering'].data['MCP_GAIN'][0] # MCP gain in DN
+    gain_v = mcp_dn_to_volt(gain)  # gives Gain in Volts
+    gain_PE  = mcp_volt_to_gain(gain_v, channel="FUV")  # gives Gain in PhotoElectrons    
```

Comment:
Needs to have a check added that the `light_fits` file is not an MUV file, or pull the channel type from the `'XUV'` element of the fits file